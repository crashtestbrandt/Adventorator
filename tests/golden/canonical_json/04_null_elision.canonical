{"keep":"value","nested":{"keep_nested":"val"}}