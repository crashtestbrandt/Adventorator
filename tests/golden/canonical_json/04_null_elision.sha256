656291fb9ada1232de6f21f63529278023f90c75a7ba5f792aff613b931bdc3f
