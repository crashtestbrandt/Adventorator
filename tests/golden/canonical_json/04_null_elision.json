{
  "keep": "value",
  "remove": null,
  "nested": {
    "keep_nested": "val",
    "remove_nested": null
  }
}